File: Oke_server/oke/src/main/java/com/bnuz/oke/entity/Course.java
package com.bnuz.oke.entity;

import java.io.Serializable;
import java.util.Date;

/**
 * Course
 *
 * @author handsome
 * @date 2020年 05月15日 00:02:34
 */
public class Course implements Serializable {

	private static final long serialVersionUID = 830830966765014573L;

	private String courseNumber;

	private String courseName;

	private Date courseCreateTime;

	private Date courseEndTime;

	private Teacher teacher;

	public String getCourseNumber() {
		return courseNumber;
	}

	public void setCourseNumber(String courseNumber) {
		this.courseNumber = courseNumber;
	}

	public String getCourseName() {
		return courseName;
	}

	public void setCourseName(String courseName) {
		this.courseName = courseName;
	}

	public Date getCourseCreateTime() {
		return courseCreateTime;
	}

	public void setCourseCreateTime(Date courseCreateTime) {
		this.courseCreateTime = courseCreateTime;
	}

	public Date getCourseEndTime() {
		return courseEndTime;
	}

	public void setCourseEndTime(Date courseEndTime) {
		this.courseEndTime = courseEndTime;
	}

	public Teacher getTeacher() {
		return teacher;
	}

	public void setTeacher(Teacher teacher) {
		this.teacher = teacher;
	}

	@Override
	public String toString() {
		return "Course{" +
				"courseNumber='" + courseNumber + '\'' +
				", courseName='" + courseName + '\'' +
				", courseCreateTime=" + courseCreateTime +
				", courseEndTime=" + courseEndTime +
				", teacher=" + teacher +
				'}';
	}
}

File: Oke_client/Oke_teacher/Oke_teacher/Entity/Course.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Oke_teacher.Entity
{
    /// <summary>
    /// 课堂信息封装类
    /// </summary>
    [Serializable]
    public class Course
    {
        //课程号
        public string courseNumber { set; get; }
        //课程名
        public string courseName { set; get; }
        //课程创建时间
        public DateTime? courseCreateTime { set; get; }
        //课程结束时间
        public DateTime? courseEndTime { set; get; }
        //课程所属老师
        public Teacher teacher { set; get; }
    }
}
